api/profile/profileRouter.js

```diff
@@ -100,7 +98,7 @@ const { ops } = require('../../lib');
 router.get('/', authRequired, async (req, res) => {
   const { Email } = req.profile;
 
-  ops.getAll(res, Parents.getProfilesByEmail, 'Profile', Email);
+  crudOperationsManager.getAll(res, Parents.getProfilesByEmail, 'Profile', Email);
```

Comment:
is` crudOperationsMangager` doing the same thing as the `ops` object? you just replaced it and it has extra functionality for the weekly reset?